README.md

```diff
@@ -14,7 +14,17 @@
 
 ## 🚀 Apollo Kotlin Normalized Cache
 
-This repository hosts [Apollo Kotlin](https://github.com/apollographql/apollo-kotlin)'s new normalized cache, aiming to replace the main repository version.
+This repository hosts [Apollo Kotlin](https://github.com/apollographql/apollo-kotlin)'s new normalized cache, aiming to replace the main repository's version.
+
+Compared to the previous version, this new normalized cache brings:
+
+- [Pagination support](https://apollographql.github.io/apollo-kotlin-normalized-cache/pagination-home.html)
+- [Cache control](https://apollographql.github.io/apollo-kotlin-normalized-cache/cache-control.html) (a.k.a. Time to live or expiration)
+- [Garbage collection](https://apollographql.github.io/apollo-kotlin-normalized-cache/garbage-collection.html), and [trimming](https://apollographql.github.io/apollo-kotlin-normalized-cache/trimming.html)
+- Partial results from the cache
+- API simplifications
+- Key scope support
+- SQL cache improved performance
```

Comment:
If you're fine with maintaining 2 sources of truth, this is fine. Else, I'm okay with a simple README that just redirects to the main documentation site.